reference/docs-conceptual/developer/cmdlet/approved-verbs-for-windows-powershell-commands.md

```diff
@@ -8,11 +8,7 @@ helpviewer_keywords:
 ---
 # Approved Verbs for PowerShell Commands
 
-PowerShell uses a verb-noun pair for the names of cmdlets and for their derived Microsoft .NET
-Framework classes. For example, the `Get-Command` cmdlet provided by PowerShell is used to retrieve
-all the commands that are registered in PowerShell. The verb part of the name identifies the action
-that the cmdlet performs. The noun part of the name identifies the entity on which the action is
-performed.
+PowerShell uses a verb-noun pair for the names of cmdlets and for their derived .NET classes. The verb part of the name identifies the action that the cmdlet performs. The noun part of the name identifies the entity on which the action is performed. For example, the `Get-Command` cmdlet retrieves all the commands that are registered in PowerShell.
```

Comment:
Can you reformat this paragraph using semantic linebreaks?  This makes it easier to see the changes instead of the diff tool showing a whole paragraph changed.  The content itself looks fine, but don't want future PRs to use this as an example and have long lines.  Thanks.

Reply:
Well, alright. I guess everyone is entitled to a kink or two. Old timey line breaks coming up...

Reply:
@skycommand Our [style guide](https://docs.microsoft.com/en-us/powershell/scripting/community/contributing/powershell-style-guide?view=powershell-7#limit-line-length-to-100-characters) for PowerShell-Docs set the line length at 100 characters.

Reply:
@sdwheeler Sure. But how do you justify the following?
```markdown
|[Import](/dotnet/api/System.Management.Automation.VerbsData.Import) (ip)|Creates a resource from data that is stored in a persistent data store (such as a file) or in an interchange format. For example, the `Import-CSV` cmdlet imports data from a comma-separated value (CSV) file to objects that can be used by other cmdlets. This verb is paired with `Export`.|For this action, do not use verbs such as BulkLoad or Load.|
```
It is 423 characters long.

What's the purpose of breaking some lines at column 100 if the same document must contain lines that are 423 characters long? Certainly, it doesn't serve the purpose of "improves the readability of git diffs and history" anymore. And look a few cenimeters above this message and you see how beautifully GitHub.com has wrapped the entire line.

When I ran into stuff like this, I always have to make a judgment call, because often, it is the style guide that has problem. A year ago, some of you and your colleages rejected my PRs because I weren't using the `fwlink` convention while others accepted them. There was one poignant PR in which the reviewer nearly rejected the PR because I was not using the `%variable%` convention! Well, it's invalid in PowerShell. But no. To the reviewer, it was the globally accepted way. I don't remember how exactly I got away with it.

Reply:
Tables are an exception. That is a limitation imposed by Markdown. I wish we could limit the line length for tables, but we can't.

I can't speak to those past PRs. I wrote this style guide and there is a reason for everything in it. But it is not necessarily complete and there are exceptions and edge cases. I hope that we are being more consistent these days.